DB/src/main/java/io/deephaven/db/v2/remote/ConstructSnapshot.java

```diff
@@ -507,24 +507,59 @@ static InitialSnapshot constructInitialSnapshotInPositionSpace(final Object logI
         return snapshot;
     }
 
+    /**
+     * Create a {@link BarrageMessage snapshot} of the specified table using a set of requested columns and positions.
+     * Note that this method uses an index that is in Position space, and that it is notification-oblivious, i.e. it
+     * makes no attempt to ensure that notifications are not missed.
+     *
+     * @param logIdentityObject An object used to prepend to log rows.
+     * @param table the table to snapshot.
+     * @return a snapshot of the entire base table.
+     */
+    public static BarrageMessage constructBackplaneSnapshotInPositionSpace(final Object logIdentityObject,
+                                                                           final BaseTable table) {
+        return constructBackplaneSnapshotInPositionSpace(logIdentityObject, table, null, null, makeSnapshotControl(false, table));
+    }
+
+    /**
+     * Create a {@link BarrageMessage snapshot} of the specified table using a set of requested columns and positions.
+     * Note that this method uses an index that is in Position space, and that it is notification-oblivious, i.e. it
+     * makes no attempt to ensure that notifications are not missed.
+     *
+     * @param logIdentityObject An object used to prepend to log rows.
+     * @param table the table to snapshot.
+     * @param columnsToSerialize  A {@link BitSet} of columns to include, null for all
+     * @param positionsToSnapshot An Index of positions within the table to include, null for all
+     * @return a snapshot of the entire base table.
+     */
     public static BarrageMessage constructBackplaneSnapshotInPositionSpace(final Object logIdentityObject,
                                                                            final BaseTable table,
-                                                                           final BitSet columnsToSerialize,
-                                                                           final Index positionsToSnapshot) {
+                                                                           @Nullable final BitSet columnsToSerialize,
+                                                                           @Nullable final Index positionsToSnapshot) {
         return constructBackplaneSnapshotInPositionSpace(logIdentityObject, table, columnsToSerialize, positionsToSnapshot, makeSnapshotControl(false, table));
     }
 
+    /**
+     * Create a {@link BarrageMessage snapshot} of the specified table using a set of requested columns and positions.
+     * Note that this method uses an index that is in Position space, and that it is notification-oblivious, i.e. it
+     * makes no attempt to ensure that notifications are not missed.
```

Comment:
This isn't inherently notification-oblivious; that's up to the `SnapshotControl`.